docs/stylesheets/extra.css

```diff
@@ -0,0 +1,6 @@
+.center-badges {
+  text-align: center;
+}
+:root  > * {
+  --md-primary-fg-color:        #1A88AB;
+}
```

Comment:
It don't split the code in tables:
![Uploading image.png…]()

```suggestion
}

table td code {
  white-space: nowrap;
  word-break: keep-all;
}
```